src/helpers.js

```diff
@@ -1,6 +1,6 @@
-import useQuery from 'hooks/useQuery';
+//import useQuery from 'hooks/useQuery';
 // global helper functions
-const isDevelopmentMode = useQuery().get('development');
+const isDevelopmentMode = false;//useQuery().get('development');
```

Comment:
@jajohnso This was the line react-scripts on which react-scripts barfed. It said it was an incorrect use of hooks (NB: I think it had something to do with it being called at the top-level, outside a react object).

Reply:
I believe that.
We gotta eventually figure out how to better handle this.